src/components/TasksList/TasksList.jsx

```diff
@@ -12,8 +12,8 @@ const TasksList = ({TasksList, status, modifyBuyList}) => {
         {
             TasksList.map(item => {
                 return <TaskCard
-                    key={item.id}
-                    id={item.id}
+                    key={item._id}
+                    id={item._id}
```

Comment:
Can you please bring back item.id variable because without it some errors may occur on the other pages (in backend data has two variants of id names - id and _id). You can do it like this: {item.id || item._id}